lib/pltr/v2/reducers/beats.js

```diff
@@ -58,188 +59,190 @@ const addNodeToState = (state, bookId, position, title, parentId) => {
   }
 }
 
-const beats =
-  (dataReparires) =>
-  (state = INITIAL_STATE, action) => {
-    const actionBookId = associateWithBroadestScope(action.bookId)
+const beats = (dataReparires) => (state = INITIAL_STATE, action) => {
```

Comment:
I think that you ran an older version of prettier on this file.
The new version prefers those curried function argument groups over several lines.
(It failed on CI: https://github.com/cameronsutter/plottr_electron/pull/702/checks?check_run_id=3128366013)

Reply:
i wondered why it tried to format it this way. I'm on the latest version of prettier though (according to my vscode at least)

Reply:
@Quiescent i just had to reload vscode and it did it right